apps/coordinator/src/components/Wallet/TransactionsTab/FeeBumping/components/AccelerationModal.tsx

```diff
@@ -195,7 +199,9 @@ const AccelerationModalContent: React.FC<
     if (analysisError) {
       return (
         <ErrorDialog
-          error="Failed to load available UTXOs"
+          error={
+            analysisError ? analysisError : "Failed to load available UTXOs"
+          }
```

Comment:
this shouldn't be necessary. If there's no analysis error, then the `if` on line 199 won't return true and you'll never get here. 
```suggestion
          error={analysisError}
```